docs/topics/templates/document-attributes.adoc

```diff
@@ -68,8 +68,8 @@ endif::[]
 // Versions
 :version-fabric8-maven-plugin: 3.5.40
```

Comment:
Should probably bump this to 4.1.0, but not sure how it affects other runtimes.

Reply:
@Ladicek I'll have to check with other runtime ENG teams to ensure they all moved to 4.1.0. I'd be inclined to make that update in a separate PR. Meanwhile, I'll redefine it to `4.1.0 ` for Thorntail.

Reply:
Works for me, thanks :-)